definitions/ext-argocd/definition.yml

```diff
@@ -0,0 +1,65 @@
+domain: EXT
+type: ARGOCD
+synthesis:
+  rules:
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_app_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_cluster_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_git_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_kubectl_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: argocd_redis_
+
+  - identifier: "label.app.kubernetes.io/name"
+    encodeIdentifierInGUID: true
+    name: "label.app.kubernetes.io/name"
+    conditions:
+    - attribute: metricName
+      prefix: grpc_server_
+
+  tags:
```

Comment:
nit here but is there any label we can use to know the type of the argo object? That way users would be able filter by apps or clusters, etc..

Is that desirable & possible?

Reply:
@naxhh The tag `namespaceName` will allow users to filter for that application but there isn't any specific dimension to use for tagging other than the ones specified.   Most users would sort by `clusterName` and `namespaceName`.  Not sure that answers your question.